docs/cli.md

```diff
@@ -152,710 +163,464 @@ keytabFile=file.keytab
-
-```bash
-gcli fileset update --name hadoop.schema.fileset --comment new_comment
-```
+`gcli fileset create --name <name> --properties <key-value pairs>`
+: Create a fileset.
 
-#### Rename a fileset
+`gcli fileset list --name hadoop.schema`
+: List filesets under the specified schema.
 
-```bash
-gcli fileset update --name hadoop.schema.fileset --rename new_name
-```
+`gcli fileset details --name <fileset>`
+: Display the details for a fileset.
 
-#### Display a fileset's properties
+`gcli fileset delete --name <fileset>`
+: Delete the specified fileset.
 
-```bash
-gcli fileset properties --name hadoop.schema.fileset 
-```
+`gcli fileset update --name <fileset> --comment "<new-comment>"`
+: Change the comment for a fileset
 
-#### Set a fileset's property
+`gcli fileset update --name <fileset> --rename <new-name>`
+: Rename an existing fileset.
 
-```bash
-gcli fileset set  --name hadoop.schema.fileset --property test --value value
-```
+`gcli fileset properties --name <fileset>`
+: List the properties for a fileset.
 
-#### Remove a fileset's property
+`gcli fileset set  --name <fileset> --property <property> --value <value>`
+: Add or update a property on a fileset.
 
-```bash
-gcli fileset remove --name hadoop.schema.fileset --property test
-```
+`gcli fileset remove --name <fileset> --property <property>`
+: Remove a property from a fileset.
 
 ### Column commands
 
-Note that some commands are not supported depending on what the database supports.
+Note that some commands are not supported on all databases.
 
-When setting the datatype of a column the following basic types are currently supported:
-null, boolean, byte, ubyte, short, ushort, integer, uinteger, long, ulong, float, double, date, time, timestamp, tztimestamp, intervalyear, intervalday, uuid, string, binary
+When setting the data type of a column, the following basic types are currently supported:
+`null`, `boolean`, `byte`, `ubyte`, `short`, `ushort`, `integer`, `uinteger`, `long`, `ulong`,
+`float`, `double`, `date`, `time`, `timestamp`, `tztimestamp`, `intervalyear`, `intervalday`,
+`uuid`, `string`, `binary`.
 
-In addition decimal(precision,scale) and varchar(length).
+In addition `decimal(precision,scale)` and `varchar(length)` are supported.
 
-#### Add a column
+The name of a column is of the format `<catalog>.<schema>.<table>.<column>`.
 
-```bash
-gcli column create --name catalog_postgres.hr.departments.value --datatype long
-gcli column create --name catalog_postgres.hr.departments.money --datatype "decimal(10,2)"
-gcli column create --name catalog_postgres.hr.departments.name --datatype "varchar(100)"
-gcli column create --name catalog_postgres.hr.departments.fullname --datatype "varchar(250)" --default "Fred Smith" --null=false
-```
+`gcli column create --name <column> --datatype long`
+: Create a column. You can specify the default value using the `--default` property,
+  and you can specify whether the column can be null using the `--null` option.
```

Comment:
Having several examples here would be helpful to a user.

Reply:
OK